init.lua

```diff
@@ -7,11 +8,15 @@ local style = require "core.style"
 local View = require "core.view"
 local Object = require "core.object"
 
+local function is_lite_xl()
+  return core.window_mode
+end
 
 -- CONSTANTS
-local PLUGIN_PATH = system.absolute_path(EXEDIR .. "/data/plugins")
-local PLUGIN_URL = "https://raw.githubusercontent.com/rxi/lite-plugins/master/README.md"
-
+local PLUGIN_PATH = system.absolute_path(USERDIR .. "/plugins")
```

Comment:
plugins are installed in `EXEDIR` in lite. Consider wrapping it in a conditional.

Reply:
Oh, of course. It did not occur to me.